vaadin-core-internal/pom.xml

```diff
@@ -43,6 +43,10 @@
             <groupId>com.vaadin</groupId>
             <artifactId>flow-lit-template</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>flow-react</artifactId>
```

Comment:
should add to the bom for the version management.

Reply:
pushed the commit

Reply:
So many places... I sort of expected it to come from the flow-bom

Reply:
one thing we can start to use in platform is import flow-bom. same as `hilla-bom`